Fix sending forwarded attachments with attached

```
[SUMMARY]
java.lang.NullPointerException: Folder name is null
com.flowcrypt.email.util.exception.ManualHandledException: Handled manually:


[STACK_TRACE]
com.flowcrypt.email.util.exception.ManualHandledException: Handled manually:
        at com.flowcrypt.email.util.exception.ExceptionUtil$Companion.handleError(ExceptionUtil.kt:30)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$downloadForwardedAtts$2.invokeSuspend(ForwardedAttachmentsDownloaderWorker.kt:165)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:39)
        at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)
Caused by: java.lang.NullPointerException: Folder name is null
        at java.util.Objects.requireNonNull(Objects.java:228)
        at com.sun.mail.imap.IMAPFolder.<init>(IMAPFolder.java:372)
        at com.sun.mail.gimap.GmailFolder.<init>(GmailFolder.java:191)
        at com.sun.mail.gimap.GmailStore.newIMAPFolder(GmailStore.java:85)
        at com.sun.mail.imap.IMAPStore.newIMAPFolder(IMAPStore.java:1787)
        at com.sun.mail.imap.IMAPStore.getFolder(IMAPStore.java:1738)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$getNewMsgState$2.invokeSuspend(ForwardedAttachmentsDownloaderWorker.kt:203)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$getNewMsgState$2.invoke(Unknown Source:8)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$getNewMsgState$2.invoke(Unknown Source:4)
        at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:89)
        at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:165)
        at kotlinx.coroutines.BuildersKt.withContext(Unknown Source:1)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker.getNewMsgState(ForwardedAttachmentsDownloaderWorker.kt:178)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker.access$getNewMsgState(ForwardedAttachmentsDownloaderWorker.kt:55)
        at com.flowcrypt.email.jetpack.workmanager.ForwardedAttachmentsDownloaderWorker$downloadForwardedAtts$2.invokeSuspend(ForwardedAttachmentsDownloaderWorker.kt:145)
        ... 8 more
```

--- a/FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/workmanager/ForwardedAttachmentsDownloaderWorker.kt
+++ b/FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/workmanager/ForwardedAttachmentsDownloaderWorker.kt
@@ -134,9 +134,9 @@ class ForwardedAttachmentsDownloaderWorker(context: Context, params: WorkerParam
           val atts = roomDatabase.attachmentDao().getAttachmentsSuspend(
             account.email,
             JavaEmailConstants.FOLDER_OUTBOX, msgEntity.uid
-          )
+          ).filter { it.isForwarded }
 
-          if (CollectionUtils.isEmpty(atts)) {
+          if (atts.isEmpty()) {
             roomDatabase.msgDao().updateSuspend(msgEntity.copy(state = MessageState.QUEUED.value))
             continue
           }

